governance.md

```diff
@@ -0,0 +1,57 @@
+# Open Journals Governance
+
+## The Project
+
+Open Journals (The Project) is a collection of open source software projects affiliated with the 501c3 [NumFOCUS Foundation](https://www.numfocus.org). The goal of The Project is to develop open source software and deploy open access academic publishing services.
```

Comment:
I think OJ of as more than "a collection of open source software projects"  Isn't it also an operational framework?  (or maybe something else that someone can express better than I am)

Reply:
I think you're right. I'm not sure what the correct phrase is though...

Reply:
Maybe "The Open Journals project develops and deploys open source software for open access publishing and enables publication credit for software and other research and educational products. It is a fiscally sponsored project of the 501(c)3 NumFOCUS."

I'm not sure if you need to put the fiscal sponsorship in the governance docs though.